CommandLine/programs/emake/link.cpp

```diff
@@ -12,7 +12,8 @@
 * Foundation, version 3 of the License, or (at your option) any later version.
 *
 * ENIGMA is distributed in the hope that it will be useful, but WITHOUT ANY
-* WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+* WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+*A
```

Comment:
This is one of the reasons I'm fearful of just running Astyle. Also the enormous amount of space it adds after newlines in parentheses.